docs/src/SUMMARY.md

```diff
@@ -9,6 +9,7 @@
     - [Connect my device to Azure IoT](./tutorials/connect-azure.md)
     - [Send Thin Edge Json data](./tutorials/send-thin-edge-data.md)
     - [Monitor my device](./tutorials/device-monitoring.md)
+    - [Manage my device software](./tutorials/software-management.md)
```

Comment:
```suggestion
[Install and manage software on my device](./tutorials/software-management.md)
```
Including the word "install" as people are more likely to search for that keyword than "manage".

Reply:
But manage means all 3 ops (install, update and remove).